core/whistle_apps-1.0.0/src/whapps_speech.erl

```diff
@@ -290,6 +354,37 @@ create_response(_Engine, {'error', _R}) ->
 create_response(_Engine, {'ibrowse_req_id', ReqID}) ->
     lager:debug("speech file streaming as ~p", [ReqID]),
     {'ok', ReqID};
+create_response(<<"voicefabric">> = _Engine, {'ok', "200", Headers, Content}) ->
+    [lager:debug("hdr: ~p", [H]) || H <- Headers],
+    lager:debug("converting media"),
+    RawFile = tmp_file_name(<<"raw">>),
+    WavFile = tmp_file_name(<<"wav">>),
+    _ = file:write_file(RawFile, Content),
+    From = "raw -r 8000 -e signed-integer -b 16",
```

Comment:
@jamesaimonetti 
 We tried it but got some problems:
1. sox cannot get file length which is important
2. sox generates warning message about [1], this messages prepends encoded data
3. sox cannot create right header for wav file because of [1]
4. port returned after 20 second timeout, I think caused by [1]